features/backend/shipments.feature

```diff
@@ -57,5 +57,5 @@ Feature: Shipments
          Then I should be on the shipment page with method "DHL"
 
     Scenario: Displaying the shipment state on details page
-        Given I am on the shipment page with method "DHL"
+         When I am on the shipment page with method "DHL"
```

Comment:
I don't think `Given / Then` workflow is correct, especially if there's `Given` included from the background.